### PR TITLE
Idva6 859

### PR DESCRIPTION
**Pull request by suvarnathekkoot-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/test/java/uk/gov/companieshouse/accounts/association/integration/UserCompanyAssociationsTest.java [hunk 1]
```diff
@@ -1436,6 +1445,24 @@ void inviteUserWhereAssociationBetweenInviteeUserEmailAndCompanyNumberDoesNotExi
...
+        mockMvc.perform( post( "/associations/invitations" )
+                        .header("X-Request-Id", "theId123")
+                        .header("Eric-identity", "9999")
+                        .header("ERIC-Identity-Type", "oauth2")
+                        .header("ERIC-Authorised-Key-Roles", "*")
+                        .contentType( MediaType.APPLICATION_JSON )
+                        .content( "{\"company_number\":\"111111\",\"invitee_email_id\":\"bruce.wayne@gotham.city\"}" ) )
+                .andExpect( status().isBadRequest() ).andReturn();
+
+        //Assertions.assertEquals( StatusEnum.CONFIRMED.getValue(), association.getStatus() );
```
Comment (kkonuganti-ch) [inline src/test/java/uk/gov/companieshouse/accounts/association/integration/UserCompanyAssociationsTest.java:1462]: why is assertion left here in code comments.

**Review by kkonuganti-ch — CHANGES_REQUESTED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by kkonuganti-ch — APPROVED** [review]

**Pull request merged**